azure_arc_k8s_jumpstart/docs/aro_terraform.md

```diff
@@ -0,0 +1,232 @@
+    ```
+    The following example output shows the resource group created successfully:
+    ```bash
+    {
+      "id": "/subscriptions/<guid>/resourceGroups/arcarodemo",
+      "location": "eastus",
+      "managedBy": null,
+      "name": "arcarodemo",
+      "properties": {
+          "provisioningState": "Succeeded"
+      },
+      "tags": null
+    }
+    ```
+
+  * **Create a virtual network**<br>
+    Azure Red Hat OpenShift clusters running OpenShift 4 require a virtual network with two empty subnets, for the master and worker nodes.
+
+    Create a new virtual network in the same resource group you created earlier.
+    ```bash
+    az network vnet create --resource-group $RESOURCEGROUP --name arcarodemo-vnet --address-prefixes 10.0.0.0/22
+    ```
+    The following example output shows the virtual network created successfully:
+    ```bash
+    {
+      "newVNet": {
+          "addressSpace": {
+          "addressPrefixes": [
+              "10.0.0.0/22"
+          ]
+          },
+          "id": "/subscriptions/<guid>/resourceGroups/arcarodemo/providers/Microsoft.Network/virtualNetworks/arcarodemo-vnet",
+          "location": "eastus",
+          "name": "arcarodemo-vnet",
+          "provisioningState": "Succeeded",
+          "resourceGroup": "arcarodemo",
+          "type": "Microsoft.Network/virtualNetworks"
+      }
+    }
+    ```
+  * Add an empty subnet for the master nodes.
+    ```bash
+    az network vnet subnet create \
+    --resource-group $RESOURCEGROUP \
+    --vnet-name arcarodemo-vnet \
+    --name master-subnet \
+    --address-prefixes 10.0.0.0/23 \
+    --service-endpoints Microsoft.ContainerRegistry
+    ```
+
+  * Add an empty subnet for the worker nodes.
+    ```bash
+    az network vnet subnet create \
+    --resource-group $RESOURCEGROUP \
+    --vnet-name arcarodemo-vnet \
+    --name worker-subnet \
+    --address-prefixes 10.0.2.0/23 \
+    --service-endpoints Microsoft.ContainerRegistry
+    ```
+  * [Disable subnet private endpoint policies](https://docs.microsoft.com/en-us/azure/private-link/disable-private-link-service-network-policy) on the master subnet. This is required to be able to connect and manage the cluster.
+    ```bash
+    az network vnet subnet update \
+    --name master-subnet \
+    --resource-group $RESOURCEGROUP \
+    --vnet-name arcarodemo-vnet \
+    --disable-private-link-service-network-policies true
+    ```
+  * Create the Openshift Container Cluster:
+    ```bash
+    az aro create \
+    --resource-group $RESOURCEGROUP \
+    --name $AROCLUSTER \
+    --vnet arcarodemo-vnet \
+    --master-subnet master-subnet \
+    --worker-subnet worker-subnet
+    ```
+    ![](../img/aro/image3.png)
+
+  * Finish Deploying ARO Cluster:<br>
+
+    <img src="../img/aro/image7.png" width="100%">
+
+    
+
+# Connecting to Azure Arc
+
+Now that you have a running ```aro``` cluster, lets connect the ARO cluster to Azure Arc by:<br>
```

Comment:
Suggestions: "Now that you have a running ARO cluster, let's connect it to Azure Arc"